src/components/Permissions/hooks/useAppsOrKonnectorsBySlug.tsx

```diff
@@ -19,7 +19,7 @@ interface useAppsOrKonnectorsBySlugReturns {
 }
 
 const useAppsOrKonnectorsBySlug = (
-  slugName: string
+  slugName: string | undefined
```

Comment:
why the slugName can be undefined ?

Reply:
In fact it's not, but I had a TS issue on the caller. I'll revert this change since we don't care ATM